src/style.css

```diff
@@ -1 +1,13 @@
-/* styles go here */
+
+.button {
+  margin-top: 20px;
+}
+
+.form-field {
+  display: block;
+  margin-bottom: 10px;
+}
+
+fieldset {
```

Comment:
it' better to style class selectors instead of tag ones

Reply:
and here

Reply:
isn' fixed